python/ql/lib/semmle/python/dataflow/new/internal/DataFlowPrivate.qll

```diff
@@ -484,8 +484,7 @@ predicate simpleLocalFlowStep(Node nodeFrom, Node nodeTo) {
  * or at runtime when callables in the module are called.
  */
 predicate simpleLocalFlowStepForTypetracking(Node nodeFrom, Node nodeTo) {
-  IncludePostUpdateFlow<PhaseDependentFlow<LocalFlow::localFlowStep/2>::step/2>::step(nodeFrom,
```

Comment:
Can you give some more details? Ah, from looking over the implementation of LocalFlow::localFlowStep ([src](https://github.com/github/codeql/blob/faaa558ed90be76b41f0fa537f8391ee8f18dd3c/python/ql/lib/semmle/python/dataflow/new/internal/DataFlowPrivate.qll#L367-L378)) that already does the PhaseDependentFlow + IncludePostUpdateFlow stuff :+1: